bcc-site/bcc/templates/part_detail.html

```diff
@@ -56,5 +71,28 @@
         </div>
     </div>
 </div>
+<br>
+<br>
+<br>
+<table class="table">
+    <thead>
+        <tr>
+            <th>Supplier</th>
+            <th>Supplier Part Number</th>
+            <th>MOQ</th>
+            <th>Unit Price</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for supplierPart in supplierParts %}
+        <tr>
+            <td>{{supplierPart.supplierName}}</td>
+            <td><a href="{{supplierPart.supplierLink}}">{{supplierPart.supplierPartNumber}}</a></td>
```

Comment:
In the link, suggest using `target="_blank"` to open the external website in a new tab/window